database/sqlite3/sqlite3_test.go

```diff
@@ -219,7 +234,18 @@ func TestDbPathOutput(t *testing.T) {
 
 	for _, tt := range pathTests {
 		t.Run(tt.name, func(t *testing.T) {
-			inputURL, _ := nurl.Parse(tt.in)
+			inputURL, err := nurl.Parse(tt.in)
+			if err != nil {
+				if strings.HasPrefix(err.Error(), tt.out) ||
```

Comment:
instead of checking for specific errors here, add the expected error to each test case. e.g. check `if err == tt.err`

Reply:
Changed the table to include an `err` field.  Updating the branch and pushing up the changes now.